nemo/nemo/utils/decorators/deprecated.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) tkornuta, NVIDIA AI Applications Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__author__ = "Tomasz Kornuta"
```

Comment:
This is nice, should we enforce this on all new files?

Reply:
This is my "standard" header, but Oleksii has some objections, guess we should discuss that...

Reply:
not sure about adding our own name to file

Reply:
any particular reason why not?

https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files

Reply:
It's not that I am against this, just not sure what/ if there are company regulations. Better check with someone who knows.